3p/3p.js

```diff
@@ -263,12 +265,20 @@ export function validateData(data, allowedFields) {
     mode: true,
     consentNotificationId: true,
   };
-  for (const field in data) {
-    if (!data.hasOwnProperty(field) ||
-        field in defaultAvailableFields) {
-      continue;
+
+  // Throw in a timeout, because we do not want to interrupt execution,
+  // because that would make each removal an instant backward incompatible
+  // change.
+  try {
```

Comment:
There shouldn't be any need for the try-catch. We can just call `rethrowAsync` in the for loop and `break`.